utils/compare_minions.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python
+
+import salt.config
+import salt.loader
+
+import json
+import json_delta
+
+try:
+    from colorama import Fore, Back, Style, init
```

Comment:
this is an "idiomatic" code snippet, which exposes the colorizing interface
I think it is better to keep it in a generic form, rather then customize for particular usage